### PR TITLE
Update to have default pageProps of {} instead of null

### PR DESCRIPTION
**Pull request by ijjk**
This fixes styles defined in `_app` with `styled-jsx` with a page that doesn't have `pageProps` since `styled-jsx` attempts to access `pageProps.className` which throws an error if `null`

### CODE REVIEW

**Review by timneutkens — COMMENTED** [review]
> since styled-jsx attempts to access pageProps.className which throws an error if null

Can you elaborate how it would do this? It sounds unlikely 🤔

**Comment by ijjk** [comment]
@timneutkens here's the snippet that causes an error. It is when you have use `styled-jsx` in `_app` when `pageProps` is null. It assumes `pageProps` is a valid object.

```js
Object(createClass["a" /* default */])(Identicons, [{
    key: "render",
    value: function render() {
      var _this$props = this.props,
          Component = _this$props.Component,
          pageProps = _this$props.pageProps;
      return external_react_default.a.createElement(components_Layout, null, external_react_default.a.createElement(Component, _extends({}, pageProps, {
        className: "jsx-1695664348" + " " + (pageProps.className != null && pageProps.className || "")
      })), external_react_default.a.createElement(style_default.a, {
        id: "1695664348"
      }, ["html{font-size:10px;}", "body{-webkit-font-smoothing:auto;}", ".identicons-page{margin:0;padding:0;color:rgba(0,0,0,0.7);}"]));
    }
  }]);
```

**Comment by timneutkens** [comment]
@ijjk what code compiled to that?

**Comment by giuseppeg** [comment]
@timneutkens https://github.com/zeit/styled-jsx/blob/00fcb7bcad9c8af91167056f93c63858d24769ba/src/_utils.js#L65

Let me know if you want me to fix this in styled-jsx.

File: test/unit/loadGetInitialProps.test.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ describe('loadGetInitialProps', () => {
 
   it('should resolve to null if getInitialProps is missing', async () => {
```
Comment (Timer) [inline test/unit/loadGetInitialProps.test.js]:
```suggestion
  it('should resolve to an empty object if getInitialProps is missing', async () => {
```

**Review by Timer — CHANGES_REQUESTED** [review]

**Review by Timer — APPROVED** [review]
I'm OK with this, it's similar to React's default behavior of always having an empty object. It'll also prevent oopsies since a lack of `getInitialProps` is going to be more common with a custom App.

**Pull request merged**